code/modules/mod/mod_types.dm

```diff
@@ -54,6 +54,7 @@
 		/obj/item/mod/module/flashlight,
 		/obj/item/mod/module/magboot,
 		/obj/item/mod/module/tether,
+		/obj/item/mod/module/drone // I'm lazy for testing, yell if this stays in
```

Comment:
AAAAAAAAAAAAAAAAAAAAAAA

Reply:
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

Reply:
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

(It'll stay until all the kinks are worked out)